### PR TITLE
Fixed custom font for drop down menus

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

**Comment by NSElvis** [comment]
Wow, I didn't knew this was needed. It's a bug on UIKit? Or that's how it works?

**Comment by RamonGilabert** [comment]
My guess is that you were calling the setter of set font, because no method was there before I added it!

**Comment by zenangst** [comment]
Okay to merge?

File: Source/FORMDefaultStyle.m [hunk 1]
```diff
@@ -64,7 +64,7 @@ + (void)applyStyle
     [[FORMTextField appearance] setInvalidBackgroundColor:[UIColor colorFromHex:@"FFD7D7"]];
     [[FORMTextField appearance] setInvalidBorderColor:[UIColor colorFromHex:@"EC3031"]];
 
-    [[FORMFieldValueLabel appearance] setFont:[UIFont fontWithName:@"DIN-Regular" size:15.0]];
+    [[FORMFieldValueLabel appearance] setCustomFont:[UIFont fontWithName:@"DIN-Regular" size:15.0]];
     [[FORMFieldValueLabel appearance] setTextColor:[UIColor colorFromHex:@"455C73"]];
```
Comment (NSElvis) [inline Source/FORMDefaultStyle.m]: @RamonGilabert Do we need the same for setTextColor?

**Comment by zenangst** [comment]
Borders on disabled fields are missing.

**Comment by NSElvis** [comment]
@zenangst they look better that way

**Comment by zenangst** [comment]
@NSElvis oh, I didn't know that it was intentional

**Comment by NSElvis** [comment]
@zenangst It wasn't

**Comment by zenangst** [comment]
But it's still a bug that they don't show up? They should still be configurable via `UIAppearance`, right?

**Comment by NSElvis** [comment]
@zenangst it's a bug, I'll make an issue for it

**Comment by zenangst** [comment]
Found it, I'll fix it in another pully :)

**Pull request merged**